src/psyclone/transformations.py

```diff
@@ -1082,10 +861,17 @@ class GOceanOMPLoopTrans(OMPLoopTrans):
         Loop). Actual transformation is done by
         :py:class:`base class <OMPLoopTrans>`.
 
-        :param omp_schedule: the omp schedule to be created. Must be one of
-            'runtime', 'static', 'dynamic', 'guided' or 'auto'.
+        :param str omp_directive: choose which OpenMP loop directive to use. \
+            Defaults to "do".
+        :param str omp_schedule: the OpenMP schedule to use. Must be one of \
+            'runtime', 'static', 'dynamic', 'guided' or 'auto'. Defaults to \
+            'static'.
 
         '''
+    def __init__(self, omp_directive="do", omp_schedule="static"):
```

Comment:
I'm not sure we should let the constructor set the type of OMP directive since the class assumes that it will be "do" (e.g. see the `str` method below). Perhaps just hardwire it in the call to `super()__init__`? The same goes for the earlier Dynamo-related transformations.

Reply:
For GOcean I kept the parameter and updated the str because we can generate OMP for GPU and we need the new teams distribute parallel do for targeting AMD GPUs (e.g. in Victoria's project).

For LFRic/Dynamo I took you suggestion and hardcoded the "do" to keep the previous functionality, as I am not confident all directives work and GPU offloading in not yet supported for this API.

Is this ok?

Reply:
Yep, I think that's OK.